docker-entrypoint.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 python manage.py migrate                  # Apply database migrations
-python manage.py collectstatic --noinput  # Collect static files
+
+python manage.py test
```

Comment:
Yes - you're right. The running of the tests should not be here in the first place, so I removed it. I think we probably do not even need to run tests here, since they should have already passed when the CI process runs the tests (or otherwise the code would not be merged into master).